docs/notes/2.21.x.md

```diff
@@ -0,0 +1,152 @@
+
+### Backends
+
+#### Docker
+
+The [`cache_from` field](https://www.pantsbuild.org/2.21/reference/targets/docker_image#cache_from) on `docker_image` now supports multiple values.
+
+The [new `extra_run_args` field](https://www.pantsbuild.org/2.21/reference/targets/docker_image#extra_run_args) on `docker_image` allows adding extra arguments that are passed into the `docker run ...` invocation when running the image with `pants run`.
+
+Pants now correctly extracts the image ID when using `docker buildx` version 0.13.1 or greater.
+
+#### Go
+
+Failing `go_package` tests can now be automatically retried by setting [the `[test].attempts_default` option](https://www.pantsbuild.org/2.21/docs/go#retries).
+
+Go mod files referring to Go versions with patch (like `go 1.21.0`) no longer cause Pants to crash.
+
+#### Helm
+
+Failing `helm_unittest_test` tests can now be automatically retried by setting [the `[test].attempts_default` option](https://www.pantsbuild.org/2.21/docs/helm#retries).
+
+[The `sources` field](https://www.pantsbuild.org/2.21/reference/targets/helm_chart#sources) of `helm_chart` targets now includes documentation files like `README.md` and  `values.schema.json` so that they are available once published.
+
+Versions 3.13.3 and 3.14.3 of the Helm tool are know [known](https://www.pantsbuild.org/2.21/reference/subsystems/helm#known_versions). Version 3.14.3 is now [the default version](https://www.pantsbuild.org/2.21/reference/subsystems/helm#version).
+
+The `experimental-deploy` goal now has built-in support for [the `--dry-run` option](https://www.pantsbuild.org/2.21/reference/goals/experimental-deploy#dry_run), which passes that to the underlying Helm invocation.
+
+[The `helm-k8s-parser` subsystem](https://www.pantsbuild.org/2.21/reference/subsystems/helm-k8s-parser) now uses `hikaru` version 0.16.0b0 by default. [The `helm-post-renderer` subsystem](https://www.pantsbuild.org/2.21/reference/subsystems/helm-post-renderer) now uses `yamlpath` version 3.8.1 by default.
+
+#### JavaScript
+
+Failing `javascript_test` tests can now be automatically retried by setting [the `[test].attempts_default` option](https://www.pantsbuild.org/2.21/reference/goals/test#attempts_default).
+
+`package_json` targets using yarn v2 or greater as the package manager are now supported.
+
+#### JVM
+
+Failing `kotlin_junit_test`, `junit_test`, and `scala_junit_test` tests can now be automatically retried by setting [the `[test].attempts_default` option](https://www.pantsbuild.org/2.21/docs/jvm/java-and-scala#retries).
+
+##### Scala
+
+The Scalafix can now be run by Pants using the [new `pants.backend.experimental.scala.lint.scalafix` backend](https://www.pantsbuild.org/2.21/docs/jvm/java-and-scala#fix-scala-code).
```

Comment:
Just "Scalafix" sounds better than "The Scalafix"

Reply:
Oops, I think I was meaning "The Scalafix tool" 🤷 

```suggestion
Scalafix can now be run by Pants using the [new `pants.backend.experimental.scala.lint.scalafix` backend](https://www.pantsbuild.org/2.21/docs/jvm/java-and-scala#fix-scala-code).
```